### PR TITLE
[内容修订] src/index.md

### PR DESCRIPTION
**Pull request by mia-zhao**
无关联 issues

add link for Flutter for Web developers guide

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Coming from another platform? Check out Flutter for:
 - [给 iOS SwiftUI 开发者的 Flutter 指南][SwiftUI]
 - [给 iOS UIKit 开发者的 Flutter 指南][UIKit]
 - [给 React Native 开发者的 Flutter 指南][React Native]
+- [给 Web 开发者的 Flutter 指南][Web]
```
Comment (AmosHuKe) [inline src/index.md:63]: 麻烦英文部分也补充一下：
```diff
- Coming from another platform? Check out Flutter for:
- [Android][], [SwiftUI][], [UIKit][], [React Native][], and
- [Xamarin.Forms][] developers.

+ Coming from another platform? Check out Flutter for:
+ [Android][], [SwiftUI][], [UIKit][], [React Native][], [Web][], and
+ [Xamarin.Forms][] developers.
```

**Review by AmosHuKe — COMMENTED** [review]
感谢你的贡献！😄

File: src/index.md (hunk 1 above)
Comment (mia-zhao) [reply src/index.md:63]: Hi! Just updated with the above. PTAL when you get a chance, thanks!

**Review by AmosHuKe — APPROVED** [review]
LGTM!

File: src/index.md (hunk 1 above)
Comment (AmosHuKe) [reply src/index.md:63]: Thanks for your time @mia-zhao .

**Pull request merged**